sirepo/simulation_db.py

```diff
@@ -244,13 +244,16 @@ def get_schema(sim_type):
     schema['simulationType'] = sim_type
     _SCHEMA_CACHE[sim_type] = schema
 
-    # TODO (mvk): merge common modules into app modules
+    # TODO (mvk): improve merging common and local schema
```

Comment:
funny where we disagree on spaces: #TODO(mvk) is how to do this. I'll mention here that `if()` should be `if ()` in javascript. Spaces after statement keywords and no spaces after functions in Javascript.